app/views/shared/_flash.html.erb

```diff
@@ -1,5 +1,5 @@
 <% flash.each do |msg_type, message| %>
-  <div data-controller="flash" aria-live="assertive" class="flash z-100 opacity-0">
+  <div data-controller="flash" data-flash-target="flash" aria-live="assertive" class="flash z-100 opacity-0">
```

Comment:
Lest litt på https://ux.shopify.com/building-accessible-flash-messages-72a278860572

og https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions

Fra det så virker det som vi bør være forsiktige med aria-live="assertive" - og kun bruke den om det er en feilmelding vi skal vise eller noe annet som er så tidskritisk at vi vil faktisk avbryte brukeren ved å skrike til dem. For alt annet virker det som vi bør bruke aria-live="polite" for å sikre best mulig opplevelse. 

På "assertive" så tar vi over stemmen til brukeren og avbryter hva nå enn den leste høyt nå - mens på "polite" så leser den opp etter at den er ferdig med å lese andre ting. Som det står på mozilla så bør `polite` brukes for status-meldinger (logget inn, alt som er :notice ?) og `assertive` for feilmeldinger og ting som er verdt å avbryte deg for å si.

Reply:
Det ble bra-